ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fastsync/FastSyncActions.java

```diff
@@ -218,22 +122,26 @@ public CompletableFuture<FastSyncState> downloadPivotBlockHeader(
   private CompletableFuture<FastSyncState> internalDownloadPivotBlockHeader(
       final FastSyncState currentState) {
     if (currentState.hasPivotBlockHeader()) {
+      LOG.debug("Initial sync state {} already contains the block header", currentState);
       return completedFuture(currentState);
     }
 
-    return currentState
-        .getPivotBlockHash()
-        .map(this::downloadPivotBlockHeader)
-        .orElseGet(
-            () ->
-                new PivotBlockRetriever(
-                        protocolSchedule,
-                        ethContext,
-                        metricsSystem,
-                        currentState.getPivotBlockNumber().getAsLong(),
-                        syncConfig.getFastSyncMinimumPeerCount(),
-                        syncConfig.getFastSyncPivotDistance())
-                    .downloadPivotBlockHeader());
+    return waitForPeers(1)
+        .thenCompose(
+            unused ->
+                currentState
+                    .getPivotBlockHash()
+                    .map(this::downloadPivotBlockHeader)
+                    .orElseGet(
+                        () ->
+                            new PivotBlockRetriever(
+                                    protocolSchedule,
+                                    ethContext,
+                                    metricsSystem,
+                                    currentState.getPivotBlockNumber().getAsLong(),
+                                    syncConfig.getFastSyncMinimumPeerCount(),
```

Comment:
this is going to default to 5 since we no longer override this value to 1 for PoS networks?

Reply:
on PoS this is never called, since we always have the pivot block hash